lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -219,113 +233,175 @@ define((require, exports, module) ->
       @outOfRange = not @range.contains(row, column)
 
 
+  class TokenSequenceFinder
+    constructor: (@tokenSequences, @equalTokens) ->
+      @nSequences = @tokenSequences.length
+      @currentIndices = Array(@nSequences)
+      @currentIndices.fill(0)
```

Comment:
According to the docs, [Array.fill](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/fill) is not supported in many important browsers, so I think we should avoid using it